packages/@aws-cdk/aws-apigateway/lib/model.ts

```diff
@@ -40,7 +89,107 @@ export class EmptyModel implements IModel {
  * }
  */
 export class ErrorModel implements IModel {
-    public readonly modelId = 'Error';
+  public readonly modelId = 'Error';
+}
+
+/**
+ * Interface share for creating or importing a model.
+ */
+export interface IModelConstruct extends cdk.IConstruct, IModel {
+
 }
 
-// TODO: Implement Model, enabling management of custom models.
\ No newline at end of file
+/**
+ * Model CFN resource
```

Comment:
That comment shouldn't be necessary, as the `CfnModel` class would already encapsulate this information.

Reply:
Removed the comment.